torch/distributed/_tensor/device_mesh.py

```diff
@@ -306,6 +306,7 @@ def init_device_mesh(
         >>> # xdoctest: +SKIP
         >>> from torch.distributed._tensor.device_mesh import init_device_mesh
         >>>
+        >>> one_d_mesh = init_device_mesh("cuda", mesh_shape=(8,))
```

Comment:
I think maybe `mesh_1d` and `mesh_2d` are little better names compare to `one_d_mesh` and `two_d_mesh`?